Sources/TuistSigning/SigningMatcher.swift

```diff
@@ -49,9 +49,12 @@ final class SigningMatcher: SigningMatching {
         let provisioningProfiles: [TargetName: [ConfigurationName: ProvisioningProfile]] = try signingFilesLocator.locateProvisioningProfiles(from: path)
             .map(provisioningProfileParser.parse)
             .reduce(into: [:]) { dict, profile in
-                var currentTargetDict = dict[profile.targetName] ?? [:]
-                currentTargetDict[profile.configurationName] = profile
-                dict[profile.targetName] = currentTargetDict
+                guard let path = profile.path, let (targetName, configurationName) = path.extractTargetAndConfigurationName() else {
```

Comment:
How about saving it [here](https://github.com/tuist/tuist/blob/d7f167ffa472dd304a18b47f065533b60a6d291d/Sources/TuistSigning/ProvisioningProfile/ProvisioningProfileParser.swift#L38) to `ProvisioningProfile` ?

I think we can ditch `targetName` and `configurationName` that is extracted from the profile and use the file name as source of truth